api/src/certification/complementary-certification/domain/usecases/attach-badges.js

```diff
@@ -11,23 +12,35 @@ const attachBadges = async function ({
   targetProfileIdToDetach,
   complementaryCertificationBadgesToAttachDTO,
   badgeRepository,
-  complementaryCertificationRepository,
+  complementaryCertificationForAdminRepository,
   complementaryCertificationBadgesRepository,
 }) {
   _verifyThatLevelsAreConsistent({
     complementaryCertificationBadgesToAttachDTO,
   });
 
-  await _verifyThatComplementaryCertificationExists({
-    complementaryCertificationId,
-    complementaryCertificationRepository,
-  });
-
   await _verifyThatBadgesToAttachExist({
     complementaryCertificationBadgesToAttachDTO,
     badgeRepository,
   });
 
+  const complementaryCertification = await complementaryCertificationForAdminRepository.getById({
+    complementaryCertificationId,
+  });
+
+  if (_isComplementaryCertificationHasExternalJury({ complementaryCertification })) {
```

Comment:
```suggestion
  if (complementaryCertification.hasExternalJury) {
```

Voir mon commentaire sur la fonction ci-dessous